libnavigation-base/src/main/java/com/mapbox/navigation/base/trip/model/alert/IncidentAlertType.kt

```diff
@@ -0,0 +1,69 @@
+package com.mapbox.navigation.base.trip.model.alert
+
+object IncidentAlertType {
```

Comment:
make it enum and remove int constants?

Reply:
We cannot make it an enum because we wouldn't be able to add any new types without breaking semver.

Reply:
😢